File: EasyImport.WebApi/Models/OrderDetails.cs
using System;

namespace EasyImport.WebApi.Models
{
    public class OrderDetails
    {
        public int Id { get; set; }
        public string InvoiceNumber { get; set; }
        public string ReferenceNumber { get; set; }
        public int SupplierId { get; set; }
        public DateTime DeliveryDate { get; set; }
        public DateTime OrderCreated { get; set; }
        public DateTime? ArrivalDate { get; set; }
        public DateTime? OrderCompletedOn { get; set; }
        public string SupplierName { get; set; }

        public OrderItem OrderItem { get; set; }
    }
}

File: EasyImport-App/src/app/_models/OrderDetails.ts
import { OrderItem } from "./orderItem";

export interface OrderDetails {
  id: number;
  invoiceNumber: string;
  referenceNumber: string;
  deliveryDate: Date;
  supplierId: number;
  orderCreated: Date;
  arrivalDate: Date;
  orderCompletedOn: Date;
  supplierName: string;
  total: number;
  totalTaxes: number;
  cbm: number;
  orderStatusName?: string;
  orderStatusId: number;
  depositAmount?: number;
  balanceAmount?: number;
  dateDepositPaid?: Date;
  dateBalancePaid?: Date;
  depositPaid?: boolean;
  balancePaid?: boolean;
  editable?: boolean;
  TotalValueAllOrders: number;
  orderItems: OrderItem[];
}
